### ISSUE TITLE
Phone field data not sent with the ActiveCampaign integration?

### ISSUE THREAD

**Issue by thijskaspers** [verbb/formie#710]
A client of ours is reporting that Formie is no longer sending the data of Phone fields to their ActiveCampaign account. The rest of the payload is fine and all the data arrives at AC, just not the Phonenumber field.

We are using Formie 1.5.5 and Craft 3.7.28 (latest). Before we were using Formie 1.5.3 and it worked there. We are trying to figure out of this is ActiveCampaign or Formie. I see some changes have been made to the Phone field in your latest commits, so maybe something went wrong there.

Just a heads up, still debugging to get more info.

**Comment by engram-design** [comment]
Are you able to share your settings for the phone number field? Namely, if you have country code enabled, as this does change how the phone number is stored, but I can certainly double check things here.

**Comment by thijskaspers** [comment]
Sure, it's a basic phone number field with the countrycode dropdown disabled.

Furthermore:
- placeholder filled
- no default value
- its a required field
- no content encryption

Everything else is default

**Comment by engram-design** [comment]
I have a feeling I know the [issue](https://github.com/verbb/formie/commit/45acf4beef03a9940bf64f6635663f7efde3832d). Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by thijskaspers** [comment]
Thanks Josh, we've updated to dev-craft-3 and client is reporting that phonenumbers end up in AC again :)

**Comment by engram-design** [comment]
Fixed in [1.5.6](https://github.com/verbb/formie/releases/tag/1.5.6)
